### PR TITLE
Evil 13 create UI for weekly schedule

### PR DESCRIPTION
**Pull request by irosikoni**
image

### CODE REVIEW

File: src/app/calendar/_components/CalendarReservationForm.tsx [hunk 1]
```diff
@@ -1,9 +1,8 @@
 "use client";
 import { Dropdown } from "@/components/Dropdown";
-import { Reservation } from "../page";
+import { Classroom } from "../page";
 import { useState } from "react";
 import { OrangeButton } from "@/components/OrangeButton";
```
Comment (wleklinskimateusz) [inline src/app/calendar/_components/CalendarReservationForm.tsx:5]: Add empty line after imports

File: src/app/calendar/_components/UserReservationItem.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  cancelReservation: () => void;
+  editReservation: () => void;
+}) {
+  return (
+    <div className="relative flex w-full flex-col items-center justify-between gap-3 rounded-md border-2 border-black p-2 text-center text-primary">
+      <div className="">
+        <div>{reservation.title}</div>
+        <div className="absolute right-2 top-2 z-10 flex justify-between gap-1">
+          <button
+            className="text-md h-6 w-6 cursor-pointer rounded-md p-0 text-accent"
```
Comment (wleklinskimateusz) [inline src/app/calendar/_components/UserReservationItem.tsx:18]: add aria-label= to make it accesible

File: src/app/calendar/_components/UserReservationItem.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            className="text-md h-6 w-6 cursor-pointer rounded-md p-0 text-red-500"
+            onClick={cancelReservation}
+          >
+            &#x1F5D1;
+          </button>
+        </div>
+      </div>
+      <div className="flex items-center justify-between gap-5">
+        <div className="text-center">
+          {new Date(reservation.startTime)
```
Comment (wleklinskimateusz) [inline src/app/calendar/_components/UserReservationItem.tsx]: I believe here you can just use 
```typescript
new Intl.DateTimeFormat('en-US', {
	hour: "numeric",
    minute: "numeric"
  }).format(date)
```

File: tailwind.config.ts [hunk 4]
```diff
@@ -22,8 +22,11 @@ const config: Config = {
           "linear-gradient(90deg, #a3bac3 50%, #f6a200 50%, #f6a200 100%)",
       },
       colors: {
-        primary: "#002B38",
-        secondary: "#F6A200",
+        // primary: "#002B38",
```
Comment (wleklinskimateusz) [inline tailwind.config.ts]: I think you should just delete those comments, in case of needed to go back you can revert that change in the future

**Review by wleklinskimateusz — APPROVED** [review]
Few things to fix, good otherwise

**Review by MayeMam — APPROVED** [review]

**Pull request merged**